### ISSUE TITLE
Summary plugin reads wrong motd file

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#256]
```
The summary plugin reads the contents of en_US.html.dist instead of en_US.html.
```

Original issue reported on code.google.com by `marcus.b...@gmail.com` on 4 Nov 2011 at 10:03

**Comment by GoogleCodeExporter** [comment]
```
I'll change it as follows:

#1- Try to read motd according to session language
#2- Try to read motd according to config language ($rcmail_config['language'])
#3- Try to read motd en_US
#4- If everything failed, read dist file
```

Original comment by `rol...@liebl.ath.cx` on 5 Nov 2011 at 4:49
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Roundcube 0.6 - bundle v1.8
```

Original comment by `rol...@liebl.ath.cx` on 6 Nov 2011 at 6:05
- Changed state: **Fixed**
